src/views/Tasks/TaskGroup/index.jsx

```diff
@@ -310,7 +316,11 @@ export default class TaskGroup extends Component {
       <Dashboard
         helpView={<HelpView description={description} />}
         search={<Search onSubmit={this.handleTaskGroupSearchSubmit} />}>
-        <ErrorPanel error={error} warning={Boolean(taskGroup)} />
+        <ErrorPanel
+          error={error}
+          warning={Boolean(taskGroup)}
+          className={this.classes.errorPanel}
```

Comment:
We only want this class to be active when the warning prop is set. I think something like the following:

```jsx
import classNames from 'classnames';

// ...

className={classNames([classes.warningPanel]: Boolean(taskGroup))}
```

_Note: `classes` is a property of `this.props`, not `this`_

Reply:
Also, rename the class to `warningPanel`, please.